src/stories/common/ButtonWithIcon.stories.js

```diff
@@ -26,13 +26,37 @@ const Template = (args) => ({
         `<button-with-icon v-bind="args">
             <template #text>Test Button</template>
             <template #icon>
-                <fa class="text-blue-400 mr-3" size="2x" :icon="faFlag"/>
+                <fa :icon="faFlag"/>
             </template>
         </button-with-icon>`,
 });
 
 export const Primary = Template.bind({});
 Primary.args = {
     type: 'primary',
-    size: 'text-md',
+    size: 'text-sm ',
```

Comment:
With the fix I mentioned above. Please remove the space after `text-sm`. Do the same for all the following storybook examples are well please.